src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9650,8 +9621,6 @@ void ReportVRFTerminalUnit(EnergyPlusData &state, int const VRFTUNum) // index t
     bool HRHeatRequestFlag;      // - indicates TU could be in heat mode
     bool HRCoolRequestFlag;      // - indicates TU could be in cool mode
 
-    DXCoolingCoilIndex = state.dataHVACVarRefFlow->VRFTU(VRFTUNum).CoolCoilIndex;
-    DXHeatingCoilIndex = state.dataHVACVarRefFlow->VRFTU(VRFTUNum).HeatCoilIndex;
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9653]:(style),[unreadVariable],Variable 'DXCoolingCoilIndex' is assigned a value that is never used.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9654]:(style),[unreadVariable],Variable 'DXHeatingCoilIndex' is assigned a value that is never used.